index.js

```diff
@@ -41,10 +41,10 @@ module.exports = {
     if (process.env.EMBER_CLI_FASTBOOT) {
       process.env.EMBER_CLI_FASTBOOT_APP_NAME = app.name;
       app.options.autoRun = false;
+      
+      // We serve the index.html from fastboot-dist, so this has to apply to both builds
+      app.options.storeConfigInMeta = false;
```

Comment:
@tomdale i would prefer if this was loaded different, but still external to the app bundle. Otherwise we will pwn the "binary identical app.js" thing we spoke about at the last meeting.

It might be fine for now, but this should be added to the checklist of things to fix/think about.